packages/@aws-cdk/aws-rds/lib/instance-parameter-group.ts

```diff
@@ -0,0 +1,130 @@
+import { CfnOutput, Construct, IConstruct, Token } from '@aws-cdk/cdk';
+import { Parameters } from './props';
+import { CfnDBParameterGroup } from './rds.generated';
+
+/**
+ * A instance parameter group
+ */
+export interface IInstanceParameterGroup extends IConstruct {
+  /**
+   * Name of this parameter group
+   */
+  readonly parameterGroupName: string;
+
+  /**
+   * Export this parameter group
+   */
+  export(): InstanceParameterGroupImportProps;
+}
+
+/**
+ * Properties to reference a instance parameter group
+ */
+export interface InstanceParameterGroupImportProps {
+  readonly parameterGroupName: string;
+}
+
+/**
+ * Properties for a instance parameter group
+ */
+export interface InstanceParameterGroupProps {
+  /**
+   * Database family of this parameter group
+   */
+  family: string;
+
+  /**
+   * Description for this parameter group
+   */
+  description: string;
+
+  /**
+   * The parameters in this parameter group
+   */
+  parameters?: Parameters;
+}
+
+/**
+ * Defina a instance parameter group
+ */
+export class InstanceParameterGroup extends Construct implements IInstanceParameterGroup {
+  /**
+   * Import a parameter group
+   */
+  public static import(scope: Construct, id: string, props: InstanceParameterGroupImportProps): IInstanceParameterGroup {
+    return new ImportedInstanceParameterGroup(scope, id, props);
+  }
+
+  public readonly parameterGroupName: string;
+  private readonly parameters: Parameters = {};
+
+  constructor(scope: Construct, id: string, props: InstanceParameterGroupProps) {
+    super(scope, id);
+
+    const { description, family, parameters } = props;
+
+    const resource = new CfnDBParameterGroup(this, 'Resource', {
+      description,
+      family,
+      parameters: new Token(() => this.parameters)
+    });
+
+    for (const [key, value] of Object.entries(parameters || {})) {
+      this.setParameter(key, value);
+    }
+
+    this.parameterGroupName = resource.ref;
```

Comment:
There should be a safe `ref` on the `Resource`, e.g. `resource.parameterGroupName`.

Reply:
fixed 8a22d7c